config/config.yaml

```diff
@@ -110,10 +110,15 @@ nats:
   key_file: /path/to/key
   root_ca: /path/to/ca
 
-# Default policy of placing containers in NeoFS
-# If a user sends a request `CreateBucket` and doesn't define policy for placing of a container in NeoFS, the S3 Gateway
-# will put the container with default policy. It can be specified via environment variable, e.g.:
-default_policy: REP 3
+# Parameters of NeoFS container placement policy
+placement_policy:
+  # Default policy of placing containers in NeoFS
+  # If a user sends a request `CreateBucket` and doesn't define policy for placing of a container in NeoFS, the S3 Gateway
+  # will put the container with default policy.
+  default: REP 3
+  # Region to placement policy mapping json file.
+  # Path to container policy mapping. The same as '--container-policy' flag for authmate
+  region_mapping: /path/to/container/policy.json
```

Comment:
At first I thought about having mapping directly in config (see below), but I think it might provide some issues for JSON encoded policy. Also it is more consistent having the same mapping format for both authmate and gateway.

Anyway, I would like to see @realloc comment about that.

```yaml
placement_policy:
  region_mapping:
    rep3: REP 3
    complex: REP 1 IN X CBF 1 SELECT 1 FROM * AS X
```

Reply:
> At first I thought about having mapping directly in config

Me too. But in such case we couldn't set such mapping using env variables. (They cannot be fetched using `viper.GetStringMapString()`)